analyze_hc_restore.pl

```diff
@@ -1574,10 +1580,17 @@ sub rem_cmd_line_switch
         exit (1);
       }
 
-      my @options = ("--gpu-loops", "-u");
+      my @options = ("-u", "--gpu-loops");
 
       add_cmd_line_param (\%file_info, \@options, $gpu_loops_param);
     }
+    
+    if ($gpu_force eq "1")
+    {
+        print "\nWARNING: adding --force to the command line overrides warnings, do not report related errors\n";
```

Comment:
intentation (spaces at the start) seems wrong here

Reply:
Hmm, although I was initially getting spacing errors thrown here as well, I had thought they were resolved but will revisit this.

Reply:
resolved in updated commit [9f7a9e5](https://github.com/philsmd/analyze_hc_restore/pull/3/commits/9f7a9e590fd748e6f2cb50ba1d10fe3dda555c1e)